edge/cmd/edge_core.go

```diff
@@ -10,6 +10,7 @@ import (
 	_ "github.com/kubeedge/kubeedge/edge/pkg/metamanager"
 	_ "github.com/kubeedge/kubeedge/edge/pkg/servicebus"
 	_ "github.com/kubeedge/kubeedge/edge/test"
+	_ "github.com/kubeedge/kubeedge/tests/stubs/edge/handlerstub"
```

Comment:
Can all the source code changes required for testing be handled by a preRun script ? It increases binary size

Reply:
@sids-b Thanks your suggestion. I will remove it here.

Reply:
@sids-b
Current the references in cloud and edge are removed.
I will add the script into the performance test.
If it it possible, I pefer another PR. Thanks.